examples/create-proposal/index.js

```diff
@@ -15,15 +15,15 @@ async function main() {
   const proposal = await client.proposal.create({
     proposal: {
       contract: {
-        address: '0x76d1d77e2b571f332e1128bDd6640C68aa3d136d',
+        address: '0x63966e9D09bf401106c515E71A426A160ee5C771',
```

Comment:
Deployed a new example contract in Sepolia, previous address was for Goerli network